recipes/bzip3/all/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.4)
+project(bzip3 LANGUAGES C)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(KEEP_RPATHS)
+
+option(WITH_THREAD "enable thread" OFF)
+option(WITH_UTIL "build and install utility" OFF)
+
+set(BZIP3_SRC
+    source_subfolder/src/cm.c
+    source_subfolder/src/crc32.c
+    source_subfolder/src/libbz3.c
+    source_subfolder/src/libsais.c
+    source_subfolder/src/lzp.c
+    source_subfolder/src/rle.c
+)
+
+set(BZIP3_INC
+    source_subfolder/include/libbz3.h
+)
+
+include_directories(source_subfolder/include)
```

Comment:
Should the `target`-based from of `include_directories(source_subfolder/include)` be preferred here, i.e.  `target_include_directories(bzip3 PUBLIC source_subfolder/include)`?

Reply:
Thank you for review.
I missed this feature. I fixed it.
Please review again.